### ISSUE TITLE
Launcher bug when launching Tekkit

### ISSUE THREAD

**Issue by ashleydavies** [TechnicPack/TechnicLauncher#633]
http://pastebin.com/PG6fnpFU

Just doing as the launcher says and reporting the issue. Apologies in advance if it's a duplicate, pretty busy and can't check against other issues.

Occurred after clicking play or start or whatever on Tekkit Main. Have tekkit main installed already. Can provide any additional details if necessary. Attempting a reinstall now.

**Comment by ashleydavies** [comment]
Fixed after downloading launcher again. I'll leave this issue open in-case it helps solve a bug in the code or something. Not sure who gets to close issues since I don't open many; if I should close just notify me.

**Comment by CannibalVox** [comment]
You're fine, thank you for both of these.  I've definitely never seen this one before.  How long did you have the launcher before redownloading?

**Comment by ashleydavies** [comment]
Judging by .technic folder's creation date, 31/12/2013, so roughly 5 months. If it helps, the last time I closed Tekkit before this happened I forced it closed with task manager because it wasn't responding.

Also, I tried it after resetting the install on the launcher by clicking the cog under Tekkit and clicking reset; not sure precisely what that does but I figured it wiped the files, if so, it must've been the launcher download, if not, my Tekkit install might've been damaged possibly?
